app/javascript/src/components/Dashboard/Notes/Note/Footer.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import { Clock } from "neetoicons";
+import { Typography, Button, Avatar } from "neetoui";
+
+const Footer = ({ createdAt }) => (
+  <div className="flex flex-row items-center justify-between">
+    <Button
+      className="text-gray-400"
+      label="Getting Started"
+      style="secondary"
+    />
+    <div className="flex flex-row gap-2 text-gray-400">
+      <Clock />
+      <Typography style="body2">{createdAt}</Typography>
```

Comment:
Should createdAt be displayed in the default format over here?

Reply:
I've changed this format.